test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,30 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
These tests can be separated into multiple more specific methods. Refer to https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html for our standard of naming the methods.